frontend/summary/summary/RoBHeatmapPlot.js

```diff
@@ -283,6 +291,44 @@ class RoBHeatmapPlot extends D3Visualization {
             .style("fill", d => d.score_text_color)
             .text(d => d.score_text);
 
+        this.direction_up = this.cells_group
```

Comment:
i like this approach, nice job. if you wanted you could also filter this.cells_data beforehand to only include cells that have the up/down you want, and then you don't embed hidden nodes to the svg

Reply:
Done!